src/pretix/presale/templates/pretixpresale/event/fragment_product_list.html

```diff
@@ -23,14 +24,15 @@ <h3 id="category-{{ tup.0.id }}">{{ tup.0.name }}</h3>
                                 <a href="{{ item.picture.url }}" class="productpicture"
                                    data-title="{{ item.name|force_escape|force_escape }}"
                                         {# Yes, double-escape to prevent XSS in lightbox #}
-                                   data-lightbox="{{ item.id }}">
+                                   data-lightbox="{{ item.id }}"
+                                   aria-label="{% blocktrans trimmed with item=item.name %}Show full-size image of {{ item }}{% endblocktrans %}">
                                     <img src="{{ item.picture|thumb:'60x60^' }}"
                                          alt="{{ item.name }}"/>
```

Comment:
Maybe aria-hidden instead of alt-text as it doesn't really add anything?

Reply:
Hm, for a blind user, the full-size image won’t help, but for a visually impaired person, using a screenreader a full-size image might actually be very important?